src/Model/SentEmail.php

```diff
@@ -110,15 +111,18 @@ public function getSmtpInfoAttribute()
     }
 
     /**
-     * Returns the header requested from our stored header info
+     * Returns a collection of all headers requested from our stored header info
+     *
+     * @return \Illuminate\Support\Collection
      */
-    public function getHeader($key)
+    public function getAllHeaders()
     {
-        $headers = collect(preg_split("/\r\n|\n|\r/", $this->headers))
+        return collect(preg_split("/(\r\n)(?!\s)/", $this->headers))
             ->filter(function ($header) {
                 return preg_match("/:/", $header);
             })
             ->transform(function ($header) {
+                $header = Str::replace("\r\n", '', $header);
```

Comment:
This should comply with the RFC specifying how to handle long headers (RFC 2822, 2.2.3)

When unfolding, the package should only split headers with \r\n
https://datatracker.ietf.org/doc/html/rfc2822#section-2.2.3